### ISSUE TITLE
Add multiple pedestrian traffic lights for roads with multiple pedestrian lanes

### ISSUE THREAD

**Issue by VictorPhilipp** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#155]
@t1a2l: "Can you add seperate pedestrian traffic lights for different sides of big roads? like 4 lanes and up (ofcourse there should be a madian in the middle)."

**Comment by VictorPhilipp** [comment]
Thanks, I'm not sure I explanied myself correctly.

**Comment by VictorPhilipp** [comment]
Hey, is there any ETA on this feature?

**Comment by aubergine10** [comment]
Tagging: #8

**Comment by t1a2l** [comment]
Hey, is this still in the works?

**Comment by aubergine10** [comment]
Yes, we will be doing this when we overhaul the timed traffic lights UI.

**Comment by t1a2l** [comment]
@aubergine10 - I don't like to ask this as a programmer my self... but is there a time frame like an year from now or somthing like that? where is it standing in the priority of stuff? Thanks again anyway

**Comment by aubergine10** [comment]
Depends how fast we get through the UI overhauls. First the lane arrows / connectors first (#282, #391), which will likely include conflict point visualisation (#4) at some point, which in turn will lead to better default traffic lights (#5), which in turn opens the door to the timed traffic lights overhaul (#8).
